dts/bindings/pwm/nxp,kinetis-ftm.yaml

```diff
@@ -28,4 +28,9 @@ properties:
       category: required
       description: required interrupts
       generation: define
+
+"#cells":
+  - pin
```

Comment:
Wasn't sure if we want 'pin' or 'channel'?

Reply:
channel